bitnami/keydb/templates/master/statefulset.yaml

```diff
@@ -280,7 +280,7 @@ spec:
             {{- if .Values.auth.enabled }}
             - name: REDIS_USER
               value: default
-            {{- if .Values.auth.usePasswordFiles }}
+            {{- if and .Values.auth.enabled .Values.auth.usePasswordFiles }}
```

Comment:
Not needed here, since this is already inside an `auth.enabled` conditional:

```suggestion
            {{- if .Values.auth.usePasswordFiles }}
```